builtin/logical/aws/backend_test.go

```diff
@@ -1437,6 +1437,66 @@ func testAccStepReadIamGroups(t *testing.T, name string, groups []string) logica
 	}
 }
 
+func TestBackend_iamTagsCrud(t *testing.T) {
+	t.Parallel()
```

Comment:
Generally you should avoid using `t.Parallel` as it doesn't provide much value and often confuses the test output.

Reply:
Damnable copypasta

Reply:
Removed